sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -66,16 +65,17 @@ export interface ServiceBusSender {
    */
   createMessageBatch(options?: CreateMessageBatchOptions): Promise<ServiceBusMessageBatch>;
 
-  /**
-   * Opens the AMQP link to Azure Service Bus from the sender.
-   *
-   * It is not necessary to call this method in order to use the sender. It is
-   * recommended to call this before your first sendMessages() call if you
-   * want to front load the work of setting up the AMQP link to the service.
-   *
-   * @param options - Options to configure tracing and the abortSignal.
-   */
-  open(options?: OperationOptionsBase): Promise<void>;
+  // TODO: Commented out to come up with an alternative name
+  // /**
+  //  * Opens the AMQP link to Azure Service Bus from the sender.
+  //  *
+  //  * It is not necessary to call this method in order to use the sender. It is
+  //  * recommended to call this before your first sendMessages() call if you
+  //  * want to front load the work of setting up the AMQP link to the service.
+  //  *
+  //  * @param options - Options to configure tracing and the abortSignal.
+  //  */
+  // open(options?: OperationOptionsBase): Promise<void>;
```

Comment:
Should I comment out the implementation too?

Reply:
Yes, otherwise the JS users will find it!